src/test/resources/comment_extractor_tests/outputs/rest2.txt

```diff
@@ -0,0 +1,2 @@
+Some comment*/
```

Comment:
Is this an expected behavior? Shouldn't rest be always a well-defined Scala code? Maybe mixing using directives with normal comments should be disallowed?

Reply:
Yes, probably it should be. I tried to implement a support for multi-line comments but after writing testcase realized that it can be problematic. For now, let's assume it's disallowed